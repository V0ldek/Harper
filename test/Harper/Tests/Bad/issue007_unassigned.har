fun :: Integer;
fun = {
    var (x :: Integer);
    (n :: Integer) = x;
    return n;
};

main :: Integer;
main = fun;