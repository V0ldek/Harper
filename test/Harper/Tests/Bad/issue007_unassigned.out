Error: variable `x` used before it was assigned.
During evaluation of:
x
Located at line 3 column 9

Execution terminated with an error: runtime error.